### ISSUE TITLE
Add --open flag

### ISSUE THREAD

**Issue by dansteren** [lastmjs/zwitterion#89]
It would be nice if you could run `zwitterion --open` (or something similar) and it would automatically open a browser window to `localhost:8000` or wherever the site is being served. Better yet, have this be the default behavior.

**Comment by lastmjs** [comment]
Sounds reasonable to add an option for this.

**Comment by lastmjs** [comment]
I don't think it should be default behavior, because I think that would be unexpected for something that is supposed to just be a static http server, and I personally use Zwitterion and other http servers in contexts that have nothing to do with a browser
